Packs/PrismaCloud/ReleaseNotes/4_2_3.md

```diff
@@ -0,0 +1,12 @@
+
+#### Integrations
+
+##### Prisma Cloud v2
+
+- Updated the ***prisma-cloud-remediation-command-list*** and ***prisma-cloud-alert-remediate*** commands to return a readable output instead of an error.
```

Comment:
```suggestion
- Updated the following commands to return a readable output instead of an error.
   -  ***prisma-cloud-remediation-command-list*** 
   -  ***prisma-cloud-alert-remediate*** 
```